ci-operator/step-registry/hypershift/mce/agent/create/agentserviceconfig/hypershift-mce-agent-create-agentserviceconfig-ref.metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"path": "hypershift/mce/agent/create/agentserviceconfig/hypershift-mce-agent-create-agentserviceconfig-ref.yaml",
+	"owners": {
+		"approvers": [
```

Comment:
@enxebre should we (@bryan-cox and me) to be included in the Approvers/Reviewers?